test/workers/web_hook_worker_test.rb

```diff
@@ -18,15 +17,39 @@ class WebHookWorkerTest < ActiveSupport::TestCase
     assert_same_elements [SocketError, RestClient::Exception, Errno::ECONNREFUSED, Errno::ECONNRESET],  WebHookWorker::HANDLED_ERRORS
   end
 
-  test 'webhook job performs work' do
-    webhook = Factory(:webhook, :account_created_on => true, :user_created_on => true)
+  class TransactionalTest < ActiveSupport::TestCase
+    disable_transactional_fixtures!
+    
+    setup do
+      User.current = nil
+      @worker = WebHookWorker.new
+    end
+
+    teardown do
+      User.current = nil
```

Comment:
Is this necessary yet? Aren't we doing this now for all tests? 😱

Reply:
Right :)

Reply:
Also if you ever need that, just stub the method :)